Drivers/Src/LTC2315.c

```diff
@@ -56,13 +56,13 @@ uint16_t LTC2315_Read() {
     OS_ERR err;
     uint8_t rxdata[2] = {0xff, 0xff};
 
-    LTC2315_wakeup_sleep();
-
     uint8_t count = 0;
     do {
     OSSchedLock(&err);
     assertOSError(err);
 
+    LTC2315_wakeup_sleep();
```

Comment:
should the wakeup sleep be inside the do while loop? 
idk if we want to move it back without testing tho

Reply:
i think it is fine where it is for now. ideally, the do while does not while too much